examples/cylinderflow.py

```diff
@@ -124,6 +124,7 @@ def main(nelems: int, degree: int, reynolds: float, rotation: float, radius: flo
     ns.dt = timestep
     ns.DuDt_i = '(u_i - u0_i) / dt + ∇_j(u_i) u_j' # material derivative
     ns.σ_ij = '(∇_j(u_i) + ∇_i(u_j)) / Re - p δ_ij'
+    ns.ω = '∇_1(u_0) - ∇_0(u_1)'
```

Comment:
I'm not sure if it is an improvement, but you can also write this as

```python
ns.ε = function.levicivita(2)
ns.ω = ε_ij ∇_j(u_i)
```

Reply:
done